mysqld_exporter.go

```diff
@@ -56,10 +58,6 @@ var (
 		"collect.info_schema.innodb_metrics", false,
 		"Collect metrics from information_schema.innodb_metrics",
 	)
-	collectGlobalStatus = flag.Bool(
```

Comment:
Usually you'll want to confine any flag definitions to the binary package, and optimally to its main function. I'd keep those in this package.